src/graph/FetchEdgesExecutor.cpp

```diff
@@ -49,6 +49,11 @@ Status FetchEdgesExecutor::prepareClauses() {
         if (!status.ok()) {
             break;
         }
+
+        // Add SrcID, DstID, Rank before prepareYield()
+        returnColNames_.emplace_back("SrcID");
+        returnColNames_.emplace_back("DstID");
+        returnColNames_.emplace_back("Rank");
```

Comment:
I suggest we use the reserved property name, `_src`, `_dst`, `_rank`

Reply:
`_src` and ` _dst` and `_rank` are internal names

Reply:
It is, but we would better not rename for users. And users could also use the internal names in the nGQL. By the way, the storage has considered this situation, and return these value by default, please check it.

Reply:
Emm, `_src`, `_dst`, `_rank`, they are not internal names, i think. They are just reserved prop name which could be used to get the `key` of a vertex/edge. For example, people could use `<edge>._src` to get a source vertex id in an edge.